Sources/BraintreeVenmo/Public/BraintreeVenmo/BTVenmoRequest.h

```diff
@@ -34,7 +36,7 @@ typedef NS_ENUM(NSInteger, BTVenmoPaymentMethodUsage) {
 /**
  * If set to `.multiUse`, the resulting payment method will be authorized for future payments and can be vaulted. If set to `.singleUse`, the resulting payment method will be authorized for a one-time payment and cannot be vaulted.
  *
- * Defaults to `.multiUse`.
+ * Defaults to `.none`.
  */
```

Comment:
What distinctions are there between the old Venmo flow and the two flows offered via Venmo Context (multi-use & single-use)?  How can we make that clear in this docstring?

I'm a little confused on this, since I know the "old" Venmo flow had a concept of defaulting to `multi-use`.

Reply:
Maybe we can call it `.unspecified` instead of `.none`, since (as you mentioned), it will default to multi-use. The doc string could be something like:

```
If set to `.multiUse`, the resulting payment method will be authorized for future payments and can be vaulted. If set to `.singleUse`, the resulting payment method will be authorized for a one-time payment and cannot be vaulted. If not specified, the resulting payment method will be authorized for future payment and can be vaulted; however, the updated Venmo UI will not be used.
```

Reply:
Do you know what Venmo UI updates does a merchant get with the VenmoContext flow?

Reply:
There is new language and a `half sheet` style UI (think apple pay). Maybe we say something like...

>Leaving this value as `none` will result in the legacy Venmo flow. It is recommended to select `multiUse` or `singleUse` for the best customer experience.

Reply:
Good feedback, and thank you for the clarifications! See updated docstring comment.